`fgetpos64` not defined on musl targets

I'm using slang via [`slang-rs`](https://github.com/colinmarc/slang-rs) in a rust project. I have a fallback where, if slang is not available on the system, it builds from source as part of the overall build.

A user recently sent me a build error from a musl-based system:

```
/home/stacy/sources/aports/testing/shader-slang/src/slang-2025.5/source/core/slang-stream.cpp: In member function 'virtual Slang::Int64 Slang::FileStream::getPosition()':
/home/stacy/sources/aports/testing/shader-slang/src/slang-2025.5/source/core/slang-stream.cpp:196:5: error: 'fpos64_t' was not declared in this scope; did you mean 'fpos_t'?
  196 |     fpos64_t pos;
      |     ^~~~~~~~
      |     fpos_t
/home/stacy/sources/aports/testing/shader-slang/src/slang-2025.5/source/core/slang-stream.cpp:197:26: error: 'pos' was not declared in this scope; did you mean 'pow'?
  197 |     fgetpos64(m_handle, &pos);
      |                          ^~~
      |                          pow
/home/stacy/sources/aports/testing/shader-slang/src/slang-2025.5/source/core/slang-stream.cpp:197:5: error: 'fgetpos64' was not declared in this scope; did you mean 'fgetpos'?
  197 |     fgetpos64(m_handle, &pos);
      |     ^~~~~~~~~
      |     fgetpos
ninja: subcommand failed
```

It appears from googling ([example](https://gitlab.alpinelinux.org/alpine/aports/-/issues/15573)) that `fgetpos64` is not defined on musl.

--- a/source/core/slang-stream.cpp
+++ b/source/core/slang-stream.cpp
@@ -5,6 +5,7 @@
 #include "slang-io.h"
 #include "slang-process.h"
 
+#include <stdio.h>
 #include <thread>
 
 namespace Slang

